pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <name>White Source Jenkins plugin</name>
     <description>White Source external update agent for Jenkins CI</description>
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/Whitesource+Plugin/</url>
+    <url>https://plugins.jenkins.io/whitesource/</url>
```

Comment:
This change usually needs accompanying additions to the README in order to bring documentation from the [wiki archive](https://github.com/jenkins-infra/plugins-wiki-docs/tree/master/whitesource) to the README in the plugin repository.

In this case, that would mean bringing the `docs/images` directory and the README contents.